### DISCUSSION TITLE
ArgoCD SSO via OIDC and PKCE

### DISCUSSION THREAD

**Discussion by markusobsil** [argoproj/argo-cd#9086] [Q&A]
Hi, 
I am currently working on a Proof of concept with ArgoCD and want to configure SSO via OIDC. [PKCE](https://oauth.net/2/pkce/) is mandatory in this case for the OAuth Provider I have to use.
Does anyone know, if ArgoCD SSO supports PKCE? If not, is there a workaround, e.g. using OIDC with dex?

Thank you!

**Comment by rishabh625** [comment]
This could help @markusobsil ? https://argo-cd.readthedocs.io/en/stable/operator-manual/user-management/auth0/#configure-oidc-for-argocd

https://github.com/dexidp/website/blob/main/content/docs/connectors/github.md


can look at operator docs too:
https://argocd-operator.readthedocs.io/en/latest/usage/dex/
https://argocd-operator.readthedocs.io/en/latest/reference/argocd/#dex-options

**Reply by markusobsil** [reply]
Thank you very much for these links! Unfortunately no word about PKCE in the documentation.
